api.py

```diff
@@ -3,31 +3,43 @@
 """
 import random
 import string
-
+import psycopg2
 from flask import Flask, request, jsonify
+import db_operations as db
 
 app = Flask(__name__)
 
 MAX_GROUP_SIZE = 4
 POINTS_PERCENTAGE = 1
 
-# Dummy database
-groups = {}
-users = {}
-menu_items = [
-    {"item_id": 1, "name": "Coffee", "price": 2.5},
-    {"item_id": 2, "name": "Tea", "price": 2.0},
-    {"item_id": 3, "name": "Sandwich", "price": 5.0},
-]
-
-transactions = []
+def connect_to_db():
```

Comment:
pls add function arguments for db details and credentials so we can change later on

Reply:
Moved the db credentials and details to be function args. Changed the call in main() too.